package.json

```diff
@@ -53,20 +53,20 @@
     "@types/yargs": "^11.1.2",
     "awesome-typescript-loader": "^5.2.1",
     "chai": "^4.2.0",
-    "jest": "^22.4.4",
+    "jest": "^24.9.0",
```

Comment:
why do we need to upgrade all of these other dependencies?

Reply:
I had to upgrade them in conjure-typescript-runtime for sake of typescript bump but it might not be necessary

Reply:
just seems like it wasn't in #125

Reply:
ok, will revert